### PR TITLE
BB-3058: Fix elasticache provisioning inside VPC

### PR DESCRIPTION
**Pull request by giovannicimolin**
This fixes a few bugs found when deploying a production instance on a private VPC.

**Reviewers:**
- [x] @pcockwell

### CODE REVIEW

File: modules/openedx/wordpress/outputs.tf [hunk 1]
```diff
@@ -5,3 +5,7 @@ output "rds" {
 output "security_group" {
   value = aws_security_group.wordpress
 }
+
+output "security_group_id" {
+  value = aws_security_group.wordpress.id
+}
```
Comment (pcockwell) [inline modules/openedx/wordpress/outputs.tf]: This is not necessary. If you need to access the security group id from outside the module, you can simply do `module.wordpress.security_group.id`

**Review by pcockwell — CHANGES_REQUESTED** [review]
I'm kind of grappling with the naming for your `redis-memcached.tf` file... My first thought is that anyone unfamiliar that reads the code might be somewhat confused initially why redis and memcached go in the same file. My next thought was `elasticache-clusters.tf` feels a little too vague still... And `redis-memcached-elasticache-clusters.tf` feels too long, and if you add/remove anything to that file, you'd have the rename it....

I think I'm leaning towards `elasticache-clusters.tf`. What are your thoughts @giovannicimolin

File: modules/openedx/wordpress/outputs.tf (hunk 1 above)
Comment (giovannicimolin) [reply modules/openedx/wordpress/outputs.tf]: @pcockwell This is a rebase artifact. I'll remove it.

**Comment by giovannicimolin** [comment]
@pcockwell I'll go with `elasticache-clusters.tf` :)

**Review by pcockwell — APPROVED** [review]

**Pull request merged**